localstripe/resources.py

```diff
@@ -678,6 +694,28 @@ def _api_delete(cls, id):
         schedule_webhook(Event('customer.deleted', obj))
         return super()._api_delete(id)
 
+    @classmethod
+    def _api_list_sources(cls, id, object=None, limit=10):
+        print(object)
```

Comment:
Please don't include this in final code.

Reply:
Good catch!